src/Http/Controllers/TeamspeakController.php

```diff
@@ -393,7 +414,12 @@ public function getUserID() {
     }
 
     public function getRegisterUser() {
-        return view('teamspeak::register');
+        $character = CharacterSheet::find(setting('main_character_id'));
+        $corp = CorporationSheet::find($character->corporationID);
+        $ticker = $corp->ticker;
```

Comment:
What happens if the corporation sheet does not exist? 

This case must be handled. You can use the following in order to display an error :
`redirect()->back()->with('error', 'your message')`